src/Microsoft.ML.StandardTrainers/Standard/LinearModelParameters.cs

```diff
@@ -240,10 +240,12 @@ JToken ISingleCanSavePfa.SaveAsPfa(BoundPfaContext ctx, JToken input)
         bool ISingleCanSaveOnnx.SaveAsOnnx(OnnxContext ctx, string[] outputs, string featureColumn)
         {
             Host.CheckValue(ctx, nameof(ctx));
-            Host.Check(Utils.Size(outputs) == 1);
+            Host.Assert(Utils.Size(outputs) >= 1);
 
```

Comment:
Any particular reason for changing the runtime check to a debug check?

Reply:
I was trying to make it consistent with the similar FastTree assert. I changed both to check.